rust/cardano-chain-follower/src/chain_sync.rs

```diff
@@ -363,8 +368,8 @@ async fn live_sync_backfill(
 
     while let Some(block_data) = peer.blockfetch().recv_while_streaming().await? {
         // Backfilled blocks get placed in the oldest fork currently on the live-chain.
-        let block =
-            MultiEraBlock::new(cfg.chain, block_data, &previous_point, 1).with_context(|| {
+        let block = MultiEraBlock::new(cfg.chain, block_data, &previous_point, 1.into())
```

Comment:
```suggestion
        let block = MultiEraBlock::new(cfg.chain, block_data, &previous_point, Fork::BACKFILL)
```

This is a good place to use the Fork constant for "backfill" suggested above, rather than 1,